README.md

```diff
@@ -3,31 +3,59 @@
-- Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
-- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+1. Fork the RFC repo: <https://github.com/buildpacks/rfcs>
+1. Copy `0000-template.md` to `text/0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+1. Fill in RFC. Any section can be marked as "N/A" if not applicable.
+1. Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
+1. Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
 
+### Development
 Once a pull request is opened, the RFC is now in development and the following will happen:
 
-- It will be labeled and discussed on a future working group meeting. Working group meetings happen on a weekly cadence barring exceptions. Labeling marks a RFC as general or specific to a subteam.
-- The team will discuss as much as possible in the RFC pull request directly. Any outside discussion will be summarized in the comment thread.
-- When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). This is step taken when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, super majority of the team must sign off.
-- The FCP will last 7 days. If there's unanimous agreement among the team the FCP can close early.
-- For voting, the binding votes are comprised of the core team (and subteam maintainers if labeled as a subteam RFC). Acceptance requires super majority of binding votes in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome. Super majority means 2/3 or greater and no single company can have more than 50% of countable votes.
-- If no substantial new arguments or ideas are raised, the FCP will follow the outcome decided. If there are substantial new arguments, then the RFC will go back into development.
-
-Once an RFC has been accepted, the team member who merges the pull request should do the following:
-
-- Assign an id based off the pull request number.
-- Rename the file based off the id inside `text/`.
-- Create a corresponding issue in the appropriate repo.
-- Fill in the remaining metadata at the top.
-- Commit everything.
+1. The following labels will be applied as appropriate:
+ * `spec/<spec>` - For example, an RFC proposing a change to the Platform Specification will be labeled with `spec/platform`.
+ * `audience/<audience>`- For example, an RFC proposing a new pack feature for buildpack authors will be labeled with `audience/buildpack-author`
```

Comment:
do we want to specify what are valid values for `<spec>` and `<audience>`. I think `<spec>` could probably be a generic description, but I imagine `<audience>` should be defined somewhere and linked to?

Reply:
Another alternative is we can prepoulate the labels and we can easily extend without needing new PRs to the `README`.

Reply:
I don't want to have to update the readme everytime we add a new extension spec or come up with a new persona mostly because I am pretty sure we will forget to do it. So I went ahead and prepopulated some labels https://github.com/buildpacks/rfcs/labels?sort=name-asc